newscoop/src/Newscoop/GimmeBundle/EventListener/PublicResourcesListener.php

```diff
@@ -30,7 +30,7 @@ public function onRequest(GetResponseEvent $event)
     {
         $request = $event->getRequest();
         $route = $request->attributes->get('_route');
-        if (strpos($route, 'newscoop_gimme_') === false) {
+        if (strpos($route, 'newscoop_gimme_') === false || $route == 'newscoop_get_img') {
```

Comment:
what about to make it more gerneric, like place it in some separate class in one function, we would then be able to add other routes in the future if needed and we will be able to change it in one place only, not in many places